test/unit/modules/blocks/process.js

```diff
@@ -0,0 +1,356 @@
+'use strict';
+
+var expect = require('chai').expect;
+var async = require('async');
+
+var modulesLoader = require('../../../common/initModule').modulesLoader;
+var BlockLogic = require('../../../../logic/block.js');
+var VoteLogic = require('../../../../logic/vote.js');
```

Comment:
why are these capital? other vars are camelCase. I see they are used later and made camelCase...